bin/host/src/single/cli.rs

```diff
@@ -0,0 +1,282 @@
+//! This module contains all CLI-specific code for the host binary.
+
+use super::{start_server, start_server_and_native_client, LocalKeyValueStore};
+use crate::{
+    cli::parse_b256,
+    eth::OnlineBlobProvider,
+    kv::{DiskKeyValueStore, MemoryKeyValueStore, SharedKeyValueStore, SplitKeyValueStore},
+};
+use alloy_primitives::B256;
+use alloy_provider::ReqwestProvider;
+use alloy_rpc_client::RpcClient;
+use alloy_transport_http::Http;
+use anyhow::{anyhow, Result};
+use clap::{
+    builder::styling::{AnsiColor, Color, Style},
+    ArgAction, Parser,
+};
+use op_alloy_genesis::RollupConfig;
+use reqwest::Client;
+use serde::Serialize;
+use std::{path::PathBuf, sync::Arc};
+use tokio::sync::RwLock;
+use tracing::error;
+
+/// The host binary CLI application arguments.
+#[derive(Default, Parser, Serialize, Clone, Debug)]
+#[command(styles = cli_styles())]
+pub struct SingleChainHostCli {
+    /// Verbosity level (0-2)
+    #[arg(long, short, action = ArgAction::Count)]
+    pub v: u8,
```

Comment:
Doesn't the host cli itself have this field? Is this so the verbosity positioning is graceful?

Reply:
good catch - forgot to remove